components/forms/EmailForm.tsx

```diff
@@ -48,6 +65,83 @@ export const EmailForm = (props: EmailFormProps) => {
         onChange={handleEmailChange}
         onFocus={handleEmailChange}
       />
+      {isSuccessOpen && (
+  <ModalConfirmation
+    isOpen={isSuccessOpen}
+    onClose={handleCloseModal}
+    body={
+      <div
+        style={{
+          padding: '1.5rem',
+          margin: '0',
+        }}
+      >
+        <h1
+          className={`font-tuner inline-block text-3xl lg:text-3xl lg:leading-tight bg-gradient-to-br from-orange-400 via-orange-500 to-orange-600 bg-clip-text text-transparent`}
+          style={{ fontSize: '2.5rem', margin: '0' }}
+        >
+          Welcome Aboard!
+        </h1>
+        <p
+          className="text-base lg:text-md"
+          style={{ marginBottom: '1rem' }}
+        >
+          You've been added to the llama list.
+        </p>
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'flex-end',
+            alignItems: 'center',
+            margin: '0',
+            padding: '0.5rem 0',
+          }}
+        >
+          <Button color="blue" size="medium" onClick={handleCloseModal}>
+            OK
+          </Button>
+        </div>
+      </div>
+    }
+  />
+)}
+
+{isErrorOpen && (
+  <ModalConfirmation
+    isOpen={isErrorOpen}
+    onClose={handleCloseErrorModal}
```

Comment:
```suggestion
    onClose={() =>  setIsErrorOpen(false)}
```

Refactor to just use inline function